### PR TITLE
Add RTCRtpStreamStats dictionary

### PR DESCRIPTION
**Pull request by a2sheppy**
Only Firefox info is in there; everything else is `null` except IE,
which is always `false`.

### CODE REVIEW

**Comment by Elchi3** [comment]
Thanks for your PR!

The structure looks alright to me and IE being false is reasonable, but without links to bugs or anything I can't review the Firefox information. The branch name includes https://bugzilla.mozilla.org/show_bug.cgi?id=1480498, which is about some renaming, but that renaming is not at all reflected with "alternative_name" in this PR. Please elaborate and provide more information for the reviewer.

File: api/RTCRtpStreamStats.json [hunk 1]
```diff
@@ -0,0 +1,718 @@
+{
+  "api": {
+    "RTCRtpStreamStats": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpStreamStats",
+        "support": {
+          "chrome": {
+            "version_added": null
```
Comment (jpmedley) [inline api/RTCRtpStreamStats.json]: This should be false for all Chrome, Android webview, and Opera.

**Comment by a2sheppy** [comment]
@Elchi3 - So... the data I have here comes from actually going through the code on https://hg.mozilla.org/mozilla-unified/tags, manually doing a binary search through the tags for each release version of Firefox for each item until I found where the property was added. So, you know, good fun. :)

File: api/RTCRtpStreamStats.json [hunk 2]
```diff
@@ -0,0 +1,730 @@
+{
+  "api": {
+    "RTCRtpStreamStats": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: `RTCRTPStreamStats` was added in bug 902003 for Firefox 27: https://hg.mozilla.org/mozilla-central/rev/5372aea57cdb

File: api/RTCRtpStreamStats.json [hunk 3]
```diff
@@ -0,0 +1,730 @@
...
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": [
+            {
+              "version_added": "63"
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Renamed in Firefox 63 per [bug 1480498](https://bugzilla.mozilla.org/show_bug.cgi?id=1480498): https://hg.mozilla.org/mozilla-central/rev/6f781af29708

File: api/RTCRtpStreamStats.json [hunk 4]
```diff
@@ -0,0 +1,730 @@
...
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "codecId": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 902003](https://bugzilla.mozilla.org/show_bug.cgi?id=902003) for Firefox 27, as part of the original implementation of the interface: https://hg.mozilla.org/mozilla-central/rev/5372aea57cdb

File: api/RTCRtpStreamStats.json [hunk 5]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "mediaTrackId": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 902003](https://bugzilla.mozilla.org/show_bug.cgi?id=902003) for Firefox 27, as part of the original implementation of the interface: https://hg.mozilla.org/mozilla-central/rev/5372aea57cdb

File: api/RTCRtpStreamStats.json [hunk 6]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "remoteId": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 902003](https://bugzilla.mozilla.org/show_bug.cgi?id=902003) for Firefox 27, as part of the original implementation of the interface: https://hg.mozilla.org/mozilla-central/rev/5372aea57cdb

File: api/RTCRtpStreamStats.json [hunk 7]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "ssrc": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 902003](https://bugzilla.mozilla.org/show_bug.cgi?id=902003) for Firefox 27, as part of the original implementation of the interface: https://hg.mozilla.org/mozilla-central/rev/5372aea57cdb

File: api/RTCRtpStreamStats.json [hunk 8]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "isRemote": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 902003](https://bugzilla.mozilla.org/show_bug.cgi?id=902003) after the initial addition of the interface: https://bugzilla.mozilla.org/attachment.cgi?id=821837

File: api/RTCRtpStreamStats.json [hunk 9]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "kind": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 1481851](https://bugzilla.mozilla.org/show_bug.cgi?id=1481851): https://hg.mozilla.org/mozilla-central/rev/87b754ac0db0

File: api/RTCRtpStreamStats.json [hunk 10]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "mediaType": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 1055378](https://bugzilla.mozilla.org/show_bug.cgi?id=1055378): https://hg.mozilla.org/mozilla-central/rev/e98290b4d231

File: api/RTCRtpStreamStats.json [hunk 11]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "nackCount": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 1348657](https://bugzilla.mozilla.org/show_bug.cgi?id=1348657): https://hg.mozilla.org/mozilla-central/rev/ecbc128c5289

File: api/RTCRtpStreamStats.json [hunk 12]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "pliCount": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Added in [bug 1348657](https://bugzilla.mozilla.org/show_bug.cgi?id=1348657): https://hg.mozilla.org/mozilla-central/rev/ecbc128c5289

File: api/RTCRtpStreamStats.json [hunk 13]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "firCount": {
```
[the remark this answers is not in the document]
Comment (a2sheppy) [reply api/RTCRtpStreamStats.json]: There is a more recent change to this line but it appears to be whitespace only.

File: api/RTCRtpStreamStats.json [hunk 14]
```diff
@@ -0,0 +1,730 @@
...
+              "version_added": false
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": false
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Not present per current WebIDL.

File: api/RTCRtpStreamStats.json [hunk 15]
```diff
@@ -0,0 +1,730 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "sliCount": {
```
Comment (a2sheppy) [inline api/RTCRtpStreamStats.json]: Not present in latest WebIDL.

**Comment by a2sheppy** [comment]
@Elchi3 & @jpmedley - I have finished adding comments to the PR throughout the code to provide bug and changeset links for every Firefox version number specified. I did not duplicate info for both desktop and mobile as they are the same here.

**Pull request merged**